### DISCUSSION TITLE
chance of a new release?

### DISCUSSION THREAD

**Discussion by codefromthecrypt** [goreleaser/nfpm#377] [General]
I'm looking forward to the xz compression support which is on master. Is there any chance of a release soon?

**Comment by caarlos0** [comment]
Hey!

I'll probably release either tomorrow or over the weekend...

**Comment by erikgeiser** [comment]
Unfortunately the xz implementation in Go is terribly slow so don't be surprised. We're also planning to support zstd in the future which is waaay faster with the same compression performance and it's already supported by Ubuntu and Fedora.

**Reply by codefromthecrypt** [reply]
thanks for the note! In my case the uncompressed binary is 7.5M, so probably there are longer poles elsewhere. That said, I think in general a lot of binaries get beefy fast and they might want to wait for a faster algo. cheers!
